README.md

```diff
@@ -13,33 +13,64 @@
+This `holochain-rust` repository implements three distinct yet overlapping aspects of the Holochain framework.
 
-- `holochain_core_api`: the primary client wrapper crate used to instantiate and run a Holochain genome.
-- `holochain_core`: the main crate that implements the core Holochain functionality.
-- `holochain_dna`: a crate for working with holochain DNA from a package file.  Used by both holochain_core and the [packager utility](https://github.com/holochain/holochain-cmd)
-- `holochain_agent`: a crate for managing holochain agent info, including identities, keys etc..  Used by both holochain_core and other utilities.
+1. The core Holochain functionality that executes applications
+2. A wrapper used to instantiate, manage, and run applications
+3. A library and syntax for use in Rust based development of Zomes within applications
 
-We have designed Holochain applications to consist at the low-level of WebAssembly running in a virtual machine environment.  This allows us to robustly make any language that compiles to WASM available as an option for programmers to write their Holochain applications.  However each language requires a small bit of stub code to connect into the WASM runtime environment.  [`hdk-rust`](https://github.com/holochain/hdk-rust) and [`hdk-assemblyscript`](https://github.com/holochain/hdk-assemblyscript) implement the code for Rust and Assemblyscript compatibility.  We expect many more languages to be added by the community, and there is even an article on how to [write a kit for a new language](https://holochain.github.io/holochain-rust/writing_development_kit.html).
+Let's elaborate on these three aspects.
 
-## Documentation: The Book on Holochain
+### 1 - core
+The [core](./core) folder contains the code that implements the core functionality of Holochain. It is the aspect that takes in an application DNA, and an agent, and securely runs peer-to-peer applications by exposing the API functions to Zomes. It draws on other top level definitions and functions such as [dna](./dna), [cas_implementations](./cas_implementations), [agent](./agent), and [core_types](./core_types).
+
+### 2 - core api
+The first aspect only implements the logic for the execution of a single application. Meanwhile, there is a strong need to be able to load and instantiate multiple applications, and possibly even multiple instances of the same application. The [core_api](./core_api) folder contains the code that implements this capability. Even then, this code must be employed within some other context to actually run it. Such a thing is called a Holochain "container". 
```

Comment:
As I read this, it suggests that core_api implements the container, rather than is used by a container until I got to the bottom.  The core_api is just a wrapper around core to make a clean indication of what will be supported over time an allow us to change the core implementation.  I think if we could lead with that, and then explain also that we use it in our containers.

Reply:
Er... I just learned from Damien that core and core_api are going to be integrated into one crate. I suppose there's still a meaningful distinction between them though.